### PR TITLE
[es] Probabilities and Statistics

### PR DESCRIPTION
**Pull request by fordaz**
This PR includes translations of about 48% of the content.

### CODE REVIEW

**Comment by fordaz** [comment]
FYI I'm actively working on the rest of the translation, ETA early next week.

**Comment by shervinea** [comment]
Sounds great, thank you @fordaz!

**Comment by fordaz** [comment]
Just completed the rest of the sections.

**Comment by shervinea** [comment]
Great, thanks @fordaz! 

There are also [new questions](https://github.com/shervinea/cheatsheet-translation/blob/master/es/refresher-probability.md) that were recently added at the end of the list, could you also take a look at them?

**Comment by fordaz** [comment]
Absolutely, I'll go over the new questions as well.

File: es/refresher-probability.md [hunk 1]
```diff
@@ -1,347 +1,347 @@
...
 
-&#10230;
+&#10230; Axiomas de la probabilidad - Para cada evento E, denotamos P(E) como la probabilidad de que el evento E ocurra.
 
 <br>
 
 **6. Axiom 1 ― Every probability is between 0 and 1 included, i.e:**
 
-&#10230;
+&#10230; Axioma 1 - Cada probabilidad tiene valores entre 0 y 1 inclusive, esto es:
```
Comment (jfernandoghe) [inline es/refresher-probability.md]: It must be referring to a closed or inclusive group. Meaning 0 and 1 can be values of probability. Hence, I suggest:

"Cada probabilidad tiene valores inclusivos entre 0 y 1, esto es:"

File: es/refresher-probability.md [hunk 2]
```diff
@@ -1,347 +1,347 @@
...
 
-&#10230;
+&#10230; Permutación - Una permutación es un arreglo de r objetos tomados de un grupo de n objetos, en un order arbitrario. El número de estos arreglos es dado por P(n,r), definido como:
 
 <br>
 
 **10. Combination ― A combination is an arrangement of r objects from a pool of n objects, where the order does not matter. The number of such arrangements is given by C(n,r), defined as:**
 
-&#10230;
+&#10230; Combinación - Una combinación es un arreglo de r objetos tomados de un grupo de n objetos, donde el orden no importa. The número de estos arreglos es dado por C(n,r), definido como:
```
Comment (jfernandoghe) [inline es/refresher-probability.md]: El número de estos arreglos es dado por C(n,r), definido como:

File: es/refresher-probability.md [hunk 3]
```diff
@@ -1,347 +1,347 @@
...
 
-&#10230;
+&#10230; Valor Esperado y Momentos de la Distribución - Aquí están las expresiones del valor esperado E[X], valor esperado generalizado E[g(X)], k-ésimo momento E[Xk] y función característica ψ(ω) para los casos discreto y contínuo:
 
 <br>
 
 **28. Variance ― The variance of a random variable, often noted Var(X) or σ2, is a measure of the spread of its distribution function. It is determined as follows:**
 
-&#10230;
+&#10230; Varianza - La varianza de una variable aleatoria, frecuentemente denotada por Var(X) or σ2, es una medida de la dispersión de su función de distribución. Esta determinada de la siguiente manera:
```
Comment (jfernandoghe) [inline es/refresher-probability.md]: Varianza - La varianza de una variable aleatoria, frecuentemente denotada por Var(X) or σ2, es la medida de dispersión de su función de distribución.

File: es/refresher-probability.md [hunk 4]
```diff
@@ -1,347 +1,347 @@
...
 
-&#10230;
+&#10230; Sesgo - El sesgo de un estimador ^θ se define como la diferencia entre el valor esperado de la distribución de ^θ y el valor exacto, esto es:
 
 <br>
 
 **50. Remark: an estimator is said to be unbiased when we have E[^θ]=θ.**
 
-&#10230;
+&#10230; Observación: se dice que un estimador es insesgado cuando se tiene E[^θ]=θ.
```
Comment (jfernandoghe) [inline es/refresher-probability.md]: From a Mexican point of view, never heard insesgado, is commonly used just: "no sesgado"

**Comment by fordaz** [comment]
Thanks a lot @jfernandoghe for all your feedback. I've updated the file accordingly.

**Comment by shervinea** [comment]
@jfernandoghe thanks for pinging me!

@fdiazgon @alonmar: if you have some time, could you please take a look at this translation?

**Comment by fordaz** [comment]
Thanks @alonmar for your feedback. I've just updated the file accordingly.

**Comment by shervinea** [comment]
Looks like we're good to go! Thanks a lot @fordaz, @jfernandoghe and @alonmar for the awesome work!

**Pull request merged**